### PR TITLE
build fuse3 on rhel >= 6

### PR DESCRIPTION
**Pull request by DrDaveD**
[CVM-1744](https://sft.its.cern.ch/jira/browse/CVM-1744)

### CODE REVIEW

File: packaging/rpm/cvmfs-universal.spec [hunk 1]
```diff
@@ -30,7 +30,8 @@
 %define systemd_autofs_patch 1
 %endif
 
-%if 0%{?fedora} >= 29
+# fuse3 is in epel starting with epel6
+%if 0%{?fedora} >= 29 || 0%{?rhel} >= 6
```
Comment (jblomer) [inline packaging/rpm/cvmfs-universal.spec]: Shouldn't that be `el` instead of `rhel`?

Comment (DrDaveD) [reply packaging/rpm/cvmfs-universal.spec]: [Nope](https://fedoraproject.org/wiki/Packaging:DistTag#Conditionals)

Comment (jblomer) [reply packaging/rpm/cvmfs-universal.spec]: Ok, so I understand both work in the same way.  As everything else uses `el` in the spec file, for consistency reasons I'd prefer to stick to it.

Comment (DrDaveD) [reply packaging/rpm/cvmfs-universal.spec]: There is no %{el}, only individual %{el#} numbers.  When they went to the general number they used %{rhel}.  Inconsistent, but that's the way it is.
```
$ rpm --eval '%{el}'
%{el}
$ rpm --eval '%{rhel}'
6
```

Comment (jblomer) [reply packaging/rpm/cvmfs-universal.spec]: Oh, I see. Thanks!

**Pull request merged**